btas/generic/Khatri_Rao_Product.h

```diff
@@ -0,0 +1,20 @@
+#ifndef BTAS_KRP
+#define BTAS_KRP
+#include <btas/btas.h>
+namespace btas{
+	/// The khatri-rao product is an outer product of column vectors of A
```

Comment:
need to improve comments. see class `RangeNd` in `btas/range.h` for a decent documentation example (it can still be improved. I recommend that for every parameter you also specify whether it's used for data input (`\param[in]`), output (`\param[out]`), or input/output (`\param[in,out]`).

N.B. note that you first refer to arguments as A and B, but then as B and C.